libs/providers/flagd/src/lib/flagd-provider.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  EvaluationContext,
+  Provider,
+  ResolutionDetails,
+  ProviderOptions
+} from '@openfeature/nodejs-sdk';
+import {IService} from './service/IService'
+import HTTPService from './service/http/service'
+
+export interface FlagdProviderOptions extends ProviderOptions {
+  service?: IService
+}
+
+export class FlagdProvider implements Provider {
+  metadata = {
+    name: FlagdProvider.name,
+  };
+
+  private readonly service: IService
+
+  constructor(options?: FlagdProviderOptions) {
+      if (!options || options.service == undefined) {
+        this.service = new HTTPService()
+        return
+      }
+      this.service = options.service
+  }
+
+  contextTransformer = () => ({});
```

Comment:
This can be removed if it's not being used.

Reply:
removed